### PR TITLE
Update metrics storage recommendations

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1645358
- Based on https://github.com/openshift/openshift-docs/pull/13701

Including changes only from https://github.com/openshift/openshift-docs/pull/13701/files#diff-f368171fd950e046a50929f2746793a4.

### CODE REVIEW

**Comment by jboxman** [comment]
@openshift/team-documentation, PTAL; minor changes. Thanks!

File: modules/recommended-configurable-storage-technology.adoc [hunk 1]
```diff
@@ -23,14 +23,15 @@ technologies for the given {product-title} cluster application.
 .Recommended and configurable storage technology
 [options="header"]
 |===
-|Storage type |ROX footnoteref:[rox,ReadOnlyMany]|RWX footnoteref:[rwx,ReadWriteMany] |Registry|Scaled registry|Metrics|Logging|Apps
+|Storage type |ROX footnoteref:[rox,ReadOnlyMany]|RWX footnoteref:[rwx,ReadWriteMany] |Registry|Scaled registry|Metrics footnoteref:[metrics-prometheus,Prometheus is the the underlying technology used for metrics.]|Logging|Apps
```
Comment (mburke5678) [inline modules/recommended-configurable-storage-technology.adoc]: @jboxman Do these footnotes appear properly in a preview? They do not render in GitHub.

File: modules/recommended-configurable-storage-technology.adoc [hunk 2]
```diff
@@ -23,14 +23,15 @@ technologies for the given {product-title} cluster application.
...
 
 | Block
 | Yes footnoteref:[disk,This does not apply to physical disk, VM physical disk, VMDK, loopback over NFS, AWS EBS, and Azure Disk.]
 | No
 | Configurable
 | Not configurable
-| Recommended
+| Recommended footnoteref:[metrics-warning,For metrics, it is an anti-pattern to
```
Comment (mburke5678) [inline modules/recommended-configurable-storage-technology.adoc]: @jboxman I am not familiar with the term `any-pattern`. Is this the best word to use here? I don't see it used elsewhere in the docs.

File: modules/recommended-configurable-storage-technology.adoc [hunk 3]
```diff
@@ -93,22 +93,20 @@ In a scaled/HA {product-title} registry cluster deployment:
...
+* The preferred storage technology is block storage. Do not use file storage for
+metrics.
 
 [IMPORTANT]
 ====
-Testing shows issues with using the NFS server on RHEL as storage backend for
-the container image registry. This includes the Cassandra for metrics storage.
-Therefore, using NFS to back PVs used by core services is not recommended.
+Testing shows significant unrecoverable corruptions using file storage and,
+therefore, is not recommended for use.
```
Comment (mburke5678) [inline modules/recommended-configurable-storage-technology.adoc]: s/is not/it is not
It is not recommended for use when?

File: modules/recommended-configurable-storage-technology.adoc [hunk 4]
```diff
@@ -93,22 +93,20 @@ In a scaled/HA {product-title} registry cluster deployment:
...
 
-Other NFS implementations on the marketplace might not have these issues.
-Contact the individual NFS implementation vendor for more information on any
+There are file storage implementations in the marketplace might not have these
```
Comment (mburke5678) [inline modules/recommended-configurable-storage-technology.adoc]: s/might/that might 
Or: There might be file storage implementation in the marketplace that do not have these issues.

**Comment by mburke5678** [comment]
@jboxman A few nits. Otherwise LGTM, assuming the footnotes work; they do not render here.

**Comment by jboxman** [comment]
@mburke5678, thanks for the review!

File: modules/recommended-configurable-storage-technology.adoc (hunk 1 above)
Comment (jboxman) [reply modules/recommended-configurable-storage-technology.adoc]: I don't know if GitHub understands footnotes. It does render locally successfully.

File: modules/recommended-configurable-storage-technology.adoc (hunk 2 above)
Comment (jboxman) [reply modules/recommended-configurable-storage-technology.adoc]: It's referring to this I believe: https://en.wikipedia.org/wiki/Anti-pattern

**Pull request merged**